Config.md

```diff
@@ -0,0 +1,138 @@
+    joblib.Parallel()(joblib.delayed(sqrt)(i**2) for i in range(10))
+```
+
+- **NetworkX Configurations for nx-parallel**: When calling functions within `nx-parallel`, NetworkX’s configurations will override those specified by Joblib. For example, the `nx.square_clustering` function will use the `n_jobs=6` setting from `nx.config.backends.parallel`, regardless of any Joblib settings within the same context.
+
+- **Joblib Configurations for Other Code**: For any other parallel code outside of `nx-parallel`, such as a direct call to `joblib.Parallel`, the configurations specified within the Joblib context will be applied.
+
+This behavior ensures that `nx-parallel` functions consistently use NetworkX’s settings when enabled, while still allowing Joblib configurations to apply to non-NetworkX parallel tasks.
+
+**Key Takeaway**: When both systems are used together, NetworkX's configuration (`nx.config.backends.parallel`) takes precedence for `nx-parallel` functions. To avoid unexpected behavior, ensure that the `active` setting aligns with your intended configuration system.
+
+### 3.2 Key Differences
+
+- **Parameter Handling**: The main difference is how `backend_params` are passed. Since, in networkx configurations are stored as a [`@dataclass`](https://docs.python.org/3/library/dataclasses.html), we need to pass them as a dictionary, whereas in `joblib.parallel_config` you can just pass them along with the other configurations, as shown below:
+
+  ```py
+  nx.config.backends.parallel.backend_params = {"max_nbytes": None}
+  joblib.parallel_config(backend="loky", max_nbytes=None)
+  ```
+
+- **Default Behavior**: By default, `nx-parallel` looks for configs in `joblib.parallel_config` unless `nx.config.backends.parallel.active` is set to `True`.
+
+### 3.3 When Should You Use Which System?
+
+When working with multiple NetworkX backends, it's crucial to ensure compatibility among the backends to avoid conflicts between different configurations. In such cases, using NetworkX's configuration system to configure `nx-parallel` is recommended. This approach helps maintain consistency across backends. For example:
+
+```python
+nx.config.backend_priority = ["another_nx_backend", "parallel"]
+nx.config.backends.another_nx_backend.config_1 = "xyz"
+joblib.parallel_config(n_jobs=7, verbose=50)
+
+nx.square_clustering(G)
+```
+
+In this example, if `another_nx_backend` also internally utilizes `joblib.Parallel` (without exposing it to the user) within its implementation of the `square_clustering` algorithm, then the `nx-parallel` configurations set by `joblib.parallel_config` will influence the internal `joblib.Parallel` used by `another_nx_backend`. To prevent unexpected behavior, it is advisable to configure these settings through the NetworkX configuration system.
+
+If you're using `nx-parallel` independently, either the NetworkX or `joblib` configuration system can be used, depending on your preference.
```

Comment:
addressing the question of what "independently" means: one attempt
```suggestion
If your code using `nx-parallel` does not simultaneously use joblib (including inside another networkx backend) then either the `networkx.config` system or the `joblib.parallel_config` system can be used, depending on your preference.
```

Also, I think this paragraph should appear at the top of this section, as I think most users will be in this situation.

Reply:
Is there any chance of trouble when the other backend doesn't use joblib?
For example, could the config using `joblib.parallel_config` interfere in any way with nx-cugraph or graphblas-algorithms?

I *think* the only way to have a "conflict" is if the other code is using joblib too. Right?

Reply:
I think it doesn't matter what the codebases of the other backends are: the only config options nx-parallel uses are related to joblib. And the only way for another backend to be influenced by those choices is if they look at or use the config options in joblib. So only backends which use joblib will have a possible conflict over config values. 

In the interest of putting this into place, I am going to merge this PR. And I'll open another one to make further suggestions.